core-web/libs/sdk/react/src/lib/components/Container/Container.tsx

```diff
@@ -6,6 +6,36 @@ import { PageContext } from '../../contexts/PageContext';
 import { getContainersData } from '../../utils/utils';
 import { PageProviderContext } from '../PageProvider/PageProvider';
 
+const FAKE_CONTENLET = {
+    identifier: 'TEMP_EMPTY_CONTENTLET',
+    title: 'TEMP_EMPTY_CONTENTLET',
+    contentType: 'TEMP_EMPTY_CONTENTLET_TYPE',
+    inode: 'TEMPY_EMPTY_CONTENTLET_INODE',
+    widgetTitle: 'TEMP_EMPTY_CONTENTLET'
+};
+
+function getEmptyContent(contentType: string) {
+    return contentType === 'TEMP_EMPTY_CONTENTLET_TYPE' ? EmptyContainer : NoContent;
+}
```

Comment:
This is not a good approach, we should not mix the concepts of "empty container" and "there is no component for this content type".

Just render directly your fake container when `contentlet.lenght` === 0.

Make sure is very clear the intention of this here.

Reply:
Solved!